### PR TITLE
Add download log/db and stop playback

### PR DESCRIPTION
**Pull request by Hellowlol**

### CODE REVIEW

**Comment by Hellowlol** [comment]
Damn Python 3 :P

File: plexapi/media.py [hunk 1]
```diff
@@ -254,6 +254,18 @@ def _loadData(self, data):
 
 
 @utils.registerPlexObject
+class Session(PlexObject):
+    """Represents a current session."""
+    TAG = 'Session'
+
+    def _loadData(self, data):
+        print('shit')
```
Comment (Hellowlol) [inline plexapi/media.py]: Remove me.

File: plexapi/base.py [hunk 2]
```diff
@@ -440,7 +443,13 @@ def download(self, savepath=None, keep_orginal_name=False, **kwargs):
...
             filepath = utils.download(download_url, filename=filename,
-                savepath=savepath, session=self._server._session)
+                                      savepath=savepath, session=self._server._session)
             if filepath:
                 filepaths.append(filepath)
         return filepaths
+
+    def stop(self, reason=''):
+        """Stop playback for a media item."""
+        print('called stop')
```
Comment (Hellowlol) [inline plexapi/base.py]: Remove me

**Comment by pkkid** [comment]
I know pep8 says to indent this way, but I really just can't stand reading code this way. It makes my brain hurt. Can I ask we just indent 4 spaces? I'll make sure there is a flake8 file that wont complain about it.

```python
# Pep8 says this, but I absolutly hate it. :(
filepath = utils.download(download_url,
                          filename=filename,
                          savepath=savepath,
                          session=self._server._session)

# Can we please do this..
filepath = utils.download(download_url, filename=filename,
    savepath=savepath, session=self._server._session)
````

**Pull request merged**